clients/fides-js/src/components/ExperienceDescription.tsx

```diff
@@ -5,14 +5,29 @@ const TEXT_TO_LINK = "vendors page.";
 const ExperienceDescription = ({
   description,
   onVendorPageClick,
+  allowHTMLDescription = false,
 }: {
   description: string | undefined;
   onVendorPageClick?: () => void;
+  allowHTMLDescription?: boolean | null;
 }) => {
   if (!description) {
     return null;
   }
 
+  // If allowHTMLDescription is true, render rich HTML content
+  // NOTE: We sanitize these descriptions server-side when configuring the
+  // PrivacyExperience, so it's safe to trust these
+  if (allowHTMLDescription) {
+    return (
+      <div
+        className="fides-html-description"
+        // eslint-disable-next-line react/no-danger
+        dangerouslySetInnerHTML={{ __html: description }}
```

Comment:
You have to be really careful about this. I'm pretty sure I've done this securely, and have even ensured it's disabled by default to be extra cautious, so we need to specific `allowHTMLDescription: true`